ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs

```diff
@@ -406,7 +406,7 @@ public void WbProtect5()
             using (var wb = new XLWorkbook())
             {
                 var ws = wb.Worksheets.Add("Sheet1");
-                wb.Protect(true, false, "Abc@123");
+                wb.Protect("Abc@123");
```

Comment:
I think this test was indeed to test the obsolete method. We stil have to test those. It looks like you're duplicating the `WbProtect3` method here.

Reply:
So what do you suggest, new pull request with the changes to WorkbookProtection.cs, XLCellTests.cs only?